### ISSUE TITLE
[cache tagging] OPTIONS call invalidates tags

### ISSUE THREAD

**Issue by wickedOne** [FriendsOfSymfony/FOSHttpCacheBundle#279]
currently, when doing an [`OPTIONS`](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.2) call to a specific route which is configured for tagging, [this code](https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/master/EventListener/TagSubscriber.php#L82-L91) invalidates the related / generated tags.

i'm aware that you can easily circumvent this behaviour by adding a match like this to the top of your config:

``` yml
-
  match:
      path: ^/
      methods:
          - OPTIONS
```

but am wondering, looking at the specifications for the `OPTIONS` call, whether this is desired behaviour

**Comment by dbu** [comment]
hm, interesting. i guess options could usually even be cached. but probably the HTTP specification does not guarantee this, so its considered not safe.

if memory serves correctly, #69 would provide a way to solve this (see also #241) by offering more flexibility.

**Comment by ddeboer** [comment]
Fixed in #354.
